prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+     #card-deck {
+      display: flex;
+      flex-basis: calc(100%/ 3);
+      min-height: 100vh;
```

Comment:
Hi Brennon, in each of your HTML documents, try moving the `flex-basis:` property from the `#card-deck` to the `.card-wrapper`. The `flex-basis` property only works when applied to the children of an element with `display: flex`

Reply:
i think i fixed it. when i commit again is this pull request updated?
![Screen Shot 2022-06-14 at 2 21 28 PM](https://user-images.githubusercontent.com/59620542/173662933-217f70e2-ad34-4294-a483-c4b3cdfeafdc.png)
![Screen Shot 2022-06-14 at 2 24 36 PM](https://user-images.githubusercontent.com/59620542/173662971-6660ddd2-652a-4db2-8c46-414fa737fc75.png)

![Screen Shot 2022-06-14 at 1 51 15 PM](https://user-images.githubusercontent.com/59620542/173662955-29f94eff-6b43-42b9-b583-6ef14843a5b8.png)
![Screen Shot 2022-06-14 at 2 31 42 PM](https://user-images.githubusercontent.com/59620542/173663156-8ec5dd2b-be20-4caf-beb7-82c4ac38d1aa.png)